account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -64,6 +71,28 @@ def default_get(self, field_list):
         )
         return res
 
+    @api.depends(
+        "date_type",
+        "move_date",
+        "due_date",
+        "journal_ids",
+        "invoice",
+        "target_move",
+        "allow_blocked",
+        "payment_mode",
+        "partner_ids",
+    )
+    def _compute_move_line_ids_domain(self):
+        domain = self._prepare_move_line_domain()
+        self.move_line_ids_domain = domain
+        self.has_open_items = self.env["account.move.line"].search_count(domain)
```

Comment:
This one is draining too much resource while changing selection. I think you should remove it.